test/augmentation/test_auto_operation.py

```diff
@@ -109,6 +110,9 @@ def test_smoke(self, policy):
         in_tensor = torch.rand(10, 3, 50, 50, requires_grad=True)
         aug(in_tensor)
 
+    @pytest.mark.xfail(
+        torch_version() in {'1.10.2', '1.11.0', '1.12.1', '1.13.1'}, 'randomness failing into some torch versions'
```

Comment:
now that i see it's just on fp64, should we consider it just for fp64?

Reply:
Yes